build/main.mk

```diff
@@ -1,7 +1,7 @@
 ifeq ($(HAVE_POSIX),y)
-PROGRAM_NAME = xcsoar
+PROGRAM_NAME = xcsoar$(PROGRAM_NAME_SUFFIX)
```

Comment:
What's that?

Reply:
PROGRAM_NAME_SUFFIX is necessary for the linker command. For Android Legacy it is set to "_legacy".